kotlin-js/src/jsTest/kotlin/js/function/JsConstructorFunctionTest.kt

```diff
@@ -0,0 +1,20 @@
+package js.function
+
+import js.array.JsTuple
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+@OptIn(ExperimentalJsExport::class)
```

Comment:
Could you please enable it by default [here](https://github.com/JetBrains/kotlin-wrappers/blob/3148da8ee3b193aa2a31275386c86f0072317aef/buildSrc/src/main/kotlin/kotlin-conventions.gradle.kts#L39)?